src/Plugin.cpp

```diff
@@ -248,15 +268,24 @@ void Plugin::update() {
     mScreenShotRequested               = false;
   }
 
+  // Now we waited several frames. We read the pixels, encode the data as png and notify the
+  // server's worker thread that the screen shot is done.
   if (mCaptureAtFrame > 0 && mCaptureAtFrame == GetVistaSystem()->GetFrameLoop()->GetFrameCount()) {
     logger().info("Capture screenshot {}x{}; show gui: {}", mScreenShotWidth, mScreenShotHeight,
         mScreenShotGui);
 
     auto* window = GetVistaSystem()->GetDisplayManager()->GetWindows().begin()->second;
     window->GetWindowProperties()->GetSize(mScreenShotWidth, mScreenShotHeight);
-    mScreenShot.resize(mScreenShotWidth * mScreenShotHeight * 3);
+
+    std::vector<VistaType::byte> screenshot(mScreenShotWidth * mScreenShotHeight * 3);
```

Comment:
```suggestion
    std::vector<std::byte> screenshot(mScreenShotWidth * mScreenShotHeight * 3);
```

Reply:
I now use std::byte also for the data member.